packs/equipment-effects/effect-ancestral-geometry-save-type.json

```
@@ -0,0 +1,93 @@
+{
+    "_id": "Iwn7XovHu9HXp129",
+    "img": "systems/pf2e/icons/equipment/tattoos/ancestral-geometry.webp",
+    "name": "Effect: Ancestral Geometry (Save Type)",
+    "system": {
+        "badge": {
+            "evaluate": true,
+            "labels": [
+                "Fortitude",
+                "Fortitude",
+                "Reflex",
+                "Reflex",
+                "Will",
+                "Will"
```

```suggestion
                "Fortitude",
                "Reflex",
                "Will"
```

Just make the actual effect use a d3. Paizo uses d6 because they don't want to introduce weird dice to tabletop games but we don't have to bother following their madness here.